packages/search-in-workspace/src/browser/search-in-workspace-result-tree-widget.ts

```diff
@@ -88,12 +88,12 @@ export class SearchInWorkspaceResultTreeWidget extends TreeWidget {
             children: []
         };
 
-        model.onSelectionChanged(nodes => {
+        this.toDispose.push(model.onSelectionChanged(nodes => {
             const node = nodes[0];
             if (SearchInWorkspaceResultLineNode.is(node)) {
                 this.doOpen(node, true);
             }
-        });
+        }));
 
         model.onNodeRefreshed(() => this.changeEmitter.fire(this.resultTree));
```

Comment:
this event listener also should be captured, please review all new widgets for it